src/pages/reference/handlers/graphql.md

```diff
@@ -77,6 +77,179 @@ The expression inside dynamic values should be as in JS. -->
 }
 ``` -->
 
+## Fetching SDL or introspection from CDN or somewhere
+
+Let's say the introspection disabled in your production environment of your GraphQL source, and you want to provide your SDL or introspection seperately;
+
+```json
+{
+    "sources": [
+        {
+            "name": "MyGraphQLApi",
+            "handler": {
+                "graphql": {
+                    "endpoint": "https://api.github.com/graphql",
+                    "source": "https://docs.github.com/public/schema.docs.graphql",
+                    "operationHeaders": {
+                        "Authorization": "Bearer {env.GITHUB_TOKEN}"
+                    }
+                }
+            }
+        }
+    ]
+}
+```
+
+> In this case, CLI's `build` command won't save the introspection in the artifacts, so your Mesh won't start if `source` URL is down.
+
+## Local Schemas
+
+We recommend providing local schema by using the `additionalTypeDefs` and `additionalResolvers` configuration options.
+
+However, it is also possible to use a local GraphQL Schema instance as a GraphQL Mesh source, as showcased below:
+
+```json
+{
+    "sources": [
+        {
+            "name": "MyGraphQLApi",
+            "handler": {
+                "graphql": {
+                    "source": "./my-local-schema.ts"
```

Comment:
Docs do not provide any details for deploying resolvers from custom sources. Is this actually possible with Adobe's offering? I would ask the same question regarding custom request/response schemas. 

Consider adding a link to describe this procedure.